Makefile

```diff
@@ -33,6 +33,12 @@ binary:
 	gcc -shared -o ${TOOLSDIR}/binary/binarytool.so -fPIC ${TOOLSDIR}/binary/binarytool.c
 	${TOOLSDIR}/vspec2binary.py -i:./spec/VehicleSignalSpecification.id ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION).binary
 
+protobuf:
+	${TOOLSDIR}/contrib/vspec2protobuf.py -i:spec/VehicleSignalSpecification.id -I ./spec ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION).proto
+
+ocf:
```

Comment:
My understanding is that ocf does not work as of today, if so i suggest extending the comment on line 11 to state that both c and ocf are deliberately excluded from travis builds